pkg/api/v1alpha1/cluster_test.go

```diff
@@ -93,6 +93,49 @@ func TestClusterNameLength(t *testing.T) {
 	}
 }
 
+func TestUnsupportedFeatures(t *testing.T) {
+	tests := []struct {
+		name        string
+		wantCluster *Cluster
+		wantErr     error
+	}{
+		{
+			name: "SuccessTinkerbellDatacenter",
```

Comment:
```suggestion
			name: "tinkerbell config without external etcd",
```

This is how similar tests are described.